protobuf/model_config.proto

```diff
@@ -1869,6 +1869,77 @@ message ModelResponseCache
   bool enable = 1;
 }
 
+//@@
+//@@  .. cpp:var:: message ModelMetrics
+//@@
+//@@     The metrics setting of this model.
+//@@
+message ModelMetrics
+{
+  //@@
+  //@@  .. cpp:var:: message MetricControl
+  //@@
+  //@@     Override metrics settings of this model.
+  //@@
+  message MetricControl
+  {
+    //@@
+    //@@  .. cpp:var:: message MetricIdentifier
+    //@@
+    //@@     Specify metrics to be overridden with metric_option.
+    //@@
+    message MetricIdentifier
+    {
+      //@@  .. cpp:var:: string family
+      //@@
+      //@@     The name of the metric family to override
+      //@@     with the custom value.
+      //@@
+      string family = 1;
+    }
+
+    //@@  .. cpp:var:: message HistogramOptions
+    //@@
+    //@@     Histogram metrics options.
+    //@@
+    message HistogramOptions
+    {
+      //@@  .. cpp:var:: double buckets (repeated)
+      //@@
+      //@@     Repeated double type for histogram bucket boundaries.
+      //@@
+      repeated double buckets = 1;
+    }
+
+    //@@  .. cpp:var:: MetricIdentifier metric_identifier
+    //@@
+    //@@     The identifier defining metrics to be overridden with the
+    //@@     metric_options.
+    //@@
+    MetricIdentifier metric_identifier = 1;
+
+    //@@  .. cpp:var:: oneof metric_options
+    //@@
+    //@@     The value to override the metrics defined in metric_identifier.
+    //@@
+    oneof metric_options
+    {
+      //@@  .. cpp:var:: HistogramOptions histogram_options
+      //@@
+      //@@     The custom histogram options.
+      //@@
+      HistogramOptions histogram_options = 2;
+    }
+  }
+
+  //@@
+  //@@  .. cpp::var:: MetricControl metric_control (repeated)
+  //@@
+  //@@     Optional custom configuration for selected metrics.
```

Comment:
```suggestion
  //@@     Optional custom configuration for selected metrics. Application default will be used for metrics that is not specified.
```

Reply:
add pointer to a list of selected metrics

Reply:
> add pointer to a list of selected metrics

Please elaborate.

Reply:
Updated comments